### ISSUE TITLE
Configuration Failed - IPTorrents

### ISSUE THREAD

**Issue by adinunzio10** [Jackett/Jackett#1374]
I tried searching through many posts and was not able to find a solution to my issue. 

Jackett version:   0.7.1411.0

------------------------------
[log.txt](https://github.com/Jackett/Jackett/files/999803/log.txt)
-
This is my first time using Jackett. I am trying to configure it to then add to Sonarr and Radarr.

When adding IPT I get the error " Configuration failed: Found no results while trying to browse this tracker" 

I could not find anything that helped when searching, am I missing something? I have attached logs above.

**Comment by jfeulner** [comment]
Is the error when you are looking in Sonarr and Radarr, or is the error from when  you add it into Jackett?

**Comment by adinunzio10** [comment]
This error is occuring when I add the tracker to jackett. Have not been
able to add it to sonarr or radarr due to that.

**Comment by jfeulner** [comment]
Hmm, I'm on the same version and IPTorrents is alright.

Your settings look like this? May want to try changing the URLs and what not.

![image](https://cloud.githubusercontent.com/assets/654933/26039746/145b1d04-38e4-11e7-8b39-b4fa151c4af9.png)

**Comment by kaso17** [comment]
based on the logs it might have been a temporary issue on the IPT side. Is this still a problem?

**Comment by adinunzio10** [comment]
@jfeulner - Mine looks identical to yours.

@kaso17  - Still an issue today. I noticed SickRage started giving "captcha" error so I had to manually add cookie for SickRage to work. Is there a way to add cookie to Jackett? I plan to move from SickRage to Sonarr once I can get Jackett working. 

![image](https://cloud.githubusercontent.com/assets/8533159/26054910/8653e00a-393c-11e7-9dd4-0ff40573cd1c.png)
[log.txt](https://github.com/Jackett/Jackett/files/1000865/log.txt)

**Comment by kaso17** [comment]
What happens if you open https://iptorrents.com/t in your browser. According to the log the login is working fine but accessing https://iptorrents.com/t returns "0 active torrents" while it should show the latest torrents.

**Comment by adinunzio10** [comment]
Opens just fine in the browser. Only in jackett it fails. No idea why..

**Comment by kaso17** [comment]
does it show you the latest torrents too? or just the search form?

**Comment by adinunzio10** [comment]
Yeah, looks fine to me.  

![image](https://cloud.githubusercontent.com/assets/8533159/26130822/8fc3435a-3a64-11e7-9499-31f05080b908.png)

I also have no problems with SickRage. So far it seems unique to Jackett. I have since tried downgrading by reinstalling with older versions but every version I tried gave me the same error. Must be something else on my system causing issue somehow but I have no idea what it could be?

I do have my entire media server setup on a Win10 Virtual Machine.  For testing, I just tried to do it on my host machine, also running windows 10. (I don't install any of this software to my machine, only the VM, but I did for testing) and the exact same thing happens.
